docs/flows.rst

```diff
@@ -0,0 +1,645 @@
+Flow Reference
+==========================================
+CumulusCI's suite of standard flows are grouped into various categories depending on their intended purpose.
+
+
+Continuous Integration
+----------------------
+The suite of continuous integration flows are intended to execute in the context of a CI system. They typically involve the creation of an org to have tests (Apex and Robot) run against it.
+
+ci_beta
+^^^^^^^
+
+**Description:** Install the latest beta version and runs apex tests from the managed package
+
+**This is** some ``rst`` formatted text. It appears on multiple lines in the ``docs/flows.yml`` file.
+
+.. code-block:: console
+    
+    $ it even has
+    some 
+    cool
+    code blocks in it
+
+Text required before header after code block to make things work?
+subheader ***********
+Ugh. This subheader doesn't want to work :(
+
+.. code-block:: console
+
+	1) flow: install_beta
+	    1) flow: dependencies
+	        1) task: update_dependencies
+	        2) task: deploy_pre
+	    2) task: install_managed_beta
+	    3) flow: config_managed
+	        1) task: deploy_post
+	        2) task: update_admin_profile
+	    4) task: snapshot_changes
+	2) task: run_tests
+
+|
```

Comment:
Are you trying to add extra whitespace? That should be handled in the styles for the sphinx theme rather than making semantic changes to the document structure.

Reply:
Ya, I didn't like how closely the flow steps were showing next to the heading of the next flow. I'll remove these from being produced in the output.